File: Count the Elements Smaller Than the Average/count_elements_smaller_than_average.py
n = int(input("Enter the number of elements: "))
a = []
for i in range(n):
    a.append(int(input(f"Enter element {i + 1}: ")))
total = 0
for i in range(n):
    total += a[i]
average = total / n
count = 0
for i in range(n):
    if a[i] < average:
        count += 1
print("Average is:", average)
print("Count of elements smaller than average:", count)

File: Count the Elements Smaller Than the Average/count_elements_smaller_than_average.c
#include <stdio.h>
int main() {
    int n;
    printf("Enter the number of elements: ");
    scanf("%d", &n);
    int a[n];
    for (int i = 0; i < n; i++) {
        printf("Enter element %d: ", i + 1);
        scanf("%d", &a[i]);
    }
    int total = 0;
    for (int i = 0; i < n; i++) {
        total = total + a[i];
    }
    double average = (double) total / n;
    int count = 0;
    for (int i = 0; i < n; i++) {
        if (a[i] < average) {
            count++;
        }
    }
    printf("Average is: %.2f\n", average);
    printf("Count of elements smaller than average: %d\n", count);
    return 0;
}
